### ISSUE TITLE
'CorsairLightingFirmwareStorageEEPROM' does not name a type; did you mean 'CorsairLightingFirmware'?

### ISSUE THREAD

**Issue by sesheron** [Legion2/CorsairLightingProtocol#310]
**Describe the problem**
Board failed to showup in device manager, decided to plug into Arduino and see if re-uploading will work.
Getting the error about types, more errors of similar style below.

**Screenshots**

**System (please complete the following information):**
 - OS: Windows 11
 - Board: Sparkfun Pro Micro 5V
 - Device: Lighting Node PRO
 - Version of IDE: 1.8.19  (have tried multiple versions including 2.0+
 - Version of CLP: 0.15.0
 - Version of CLP Boards: 0.3.1
 - Sketch: LightingNodePRO

**Code changes**
No changes, this is code I have uploaded to the board before. Starting to think the board bricked itself.

**Error Message**
Arduino: 1.8.19 (Windows 10), Board: "CLP SparkFun Pro Micro 5V, 16 MHz, Lighting Node PRO"
LightingNodePRO:27:1: error: 'CorsairLightingFirmwareStorageEEPROM' does not name a type; did you mean 'CorsairLightingFirmware'?
 CorsairLightingFirmwareStorageEEPROM firmwareStorage;
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 CorsairLightingFirmware
LightingNodePRO:28:34: error: 'CORSAIR_LIGHTING_NODE_PRO' was not declared in this scope
 CorsairLightingFirmware firmware(CORSAIR_LIGHTING_NODE_PRO, &firmwareStorage);
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~
LightingNodePRO:28:62: error: 'firmwareStorage' was not declared in this scope
 CorsairLightingFirmware firmware(CORSAIR_LIGHTING_NODE_PRO, &firmwareStorage);
                                                              ^~~~~~~~~~~~~~~
C:\Users\Tiger\AppData\Local\Temp\arduino_modified_sketch_440420\LightingNodePRO.ino:28:62: note: suggested alternative: 'firmware'
 CorsairLightingFirmware firmware(CORSAIR_LIGHTING_NODE_PRO, &firmwareStorage);
                                                              ^~~~~~~~~~~~~~~
                                                              firmware
LightingNodePRO:29:1: error: 'FastLEDControllerStorageEEPROM' does not name a type; did you mean 'FastLEDController'?
 FastLEDControllerStorageEEPROM storage;
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 FastLEDController
LightingNodePRO:30:34: error: 'storage' was not declared in this scope
 FastLEDController ledController(&storage);
                                  ^~~~~~~
Multiple libraries were found for "CorsairLightingProtocol.h"
 Used: Z:\My Master Documents\Arduino\libraries\Corsair_Lighting_Protocol
 Not used: Z:\My Master Documents\Arduino\libraries\arduino_688345
exit status 1
'CorsairLightingFirmwareStorageEEPROM' does not name a type; did you mean 'CorsairLightingFirmware'?
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-80-g6c4433a-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=2.6.1 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-97-gc752ad5-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=3.0.0 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=esptool version=0.4.8 packager=esp8266
Index error: could not find referenced tool name=xtensa-lx106-elf-gcc version=1.20.0-26-gb404fb9-2 packager=esp8266
Index error: could not find referenced tool name=mkspiffs version=0.1.2 packager=esp8266
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-80-g6c4433a-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=2.6.1 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-97-gc752ad5-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=3.0.0 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=esptool version=0.4.8 packager=esp8266
Index error: could not find referenced tool name=xtensa-lx106-elf-gcc version=1.20.0-26-gb404fb9-2 packager=esp8266
Index error: could not find referenced tool name=mkspiffs version=0.1.2 packager=esp8266
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-80-g6c4433a-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=2.6.1 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=xtensa-esp32-elf-gcc version=1.22.0-97-gc752ad5-5.2.0 packager=esp32
Index error: could not find referenced tool name=esptool_py version=3.0.0 packager=esp32
Index error: could not find referenced tool name=mkspiffs version=0.2.3 packager=esp32
Index error: could not find referenced tool name=esptool version=0.4.8 packager=esp8266
Index error: could not find referenced tool name=xtensa-lx106-elf-gcc version=1.20.0-26-gb404fb9-2 packager=esp8266
Index error: could not find referenced tool name=mkspiffs version=0.1.2 packager=esp8266

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.

**Comment by Legion2** [comment]
In the error message it says das multiple versions of the CLP library are installed on your PC. Make sure there is only one version which is updated.

**Comment by sesheron** [comment]
Okay, I removed the one named arduino_688345. I didn't think it was a contributing factor as it was listed as "Not Used".
Looks like when I updated to .15, it made that random number folder instead of replacing the .14 one. And it did it for a bunch of other libraries. So in result, it installed updates, but kept using the old library. Deleting that folder and running all updates corrected the issue.
